AirCasting/SearchAndFollow/SearchMap/BottomCard.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 07/03/2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct BottomCardView: View {
+    @StateObject var viewModel: BottomCardViewModel
+    private var onMarkerChangeAction: ((Int) -> ())? = nil
+    
+    init(id: Int, title: String, startTime: String, endTime: String, latitude: Double, longitude: Double) {
+        _viewModel = .init(wrappedValue: .init(id: id, title: title, startTime: startTime, endTime: endTime, latitude: latitude, longitude: longitude))
+    }
+    
+    var body: some View {
+        sessionCard
+    }
+    
+    var sessionCard: some View {
+        Button {
+            viewModel.setIsModalScreenPresented()
+            onMarkerChangeAction?(viewModel.dataModel.id)
+        } label: {
+            VStack(alignment: .leading, spacing: 5) {
+                Text(viewModel.dataModel.title)
+                    .foregroundColor(.darkBlue)
+                    .multilineTextAlignment(.leading)
+                    .lineLimit(3)
+                    .minimumScaleFactor(0.7)
+                Spacer(minLength: 0.01)
+                dataAndTime
+                    .font(Fonts.regularHeading4)
+                    .foregroundColor(.aircastingGray)
+                    .minimumScaleFactor(0.1)
+                    .scaledToFit()
+            }
+        }
+        .sheet(isPresented: .init(get: {
+            viewModel.getIsModalScreenPresented()
+        }, set: { _ in
+            viewModel.setIsModalScreenPresented()
+        }), content: { viewModel.initCompleteScreen() })
```

Comment:
The `x` mark on the complete screen stopped working with this implementation, so that needs to be fixed